terraform/modules/emr/templates/emr/trigger_s3_tagger_batch_job.sh

```diff
@@ -16,7 +16,7 @@ export https_proxy
 echo "Triggering tagger with JobName $JOB_NAME and s3 prefix $S3_PREFIX"
 aws batch submit-job \
 --job-queue pdm_object_tagger \
---job-definition pdm_object_tagger_job \
+--job-definition s3_object_tagger_job \
```

Comment:
Will the old definition still be in prod after the merge? We dont push PRs to prod during the working day to stop any potential rebooting of the EMR, so the batch EMR which starts at 5'oclock will still try and call pdm_object_tagger_job.